compiler/luci/import/src/CircleImportMetadata.cpp

```diff
@@ -153,6 +202,8 @@ CircleImportMetadata::CircleImportMetadata(const luci::CircleReader &reader)
       _op_table = decoded_op_table(buffer);
     else if (meta.name.compare("ONE_source_table") == 0)
       _source_table = decoded_source_table(buffer);
+    else if (meta.name.compare("ONE_memory_plan") == 0)
+      _memory_plan_table = decoded_memory_plan(buffer);
```

Comment:
`ONE_memory_plan` and `memory_plan_table` has difference in word `table`.
I think it would be better to use similar naming, be a `plan` or `table`